docs/src/explanation/security.md

```diff
@@ -0,0 +1,65 @@
+# Security 
+
+This page provides an overview of various aspects of security to be considered
+when operating a cluster with **Canonical Kubernetes**. To consider security
+properly, this means not just aspects of Kubernetes itself, but also how and
+where it is installed and operated.
+
+A lot of important aspects of security therefore lie outside the direct scope
+of **Canonical Kubernetes**, but links for further reading
+are provided.
+
+## Security of the snap/executable
+
+As detailed in the [snap documentation][], an application installed from a snap
+is inherently more secure than a traditionally installed application.
+Snap-based applications are installed into a sandboxed, self contained
+environment which restricts its ability to interact with the rest of user
+space.
+
+## Security of the OCI images
+
+**Canonical Kubernetes** relies on OCI standard images published as `rocks` to
+deliver the services which run and facilitate the opration of the Kubernetes
+cluster. Many of the frequently used 'standard' OCI images have known security
```

Comment:
Let's maybe not start the sentence with a negative about OCI images while having just talked about the fact that Rocks are OCI images. Could we flip it around and say "OCI images give Canonical a way to [...] in ways that other commonly used OCI images cannot."